package.json

```diff
@@ -40,14 +40,14 @@
     "node": ">=4"
   },
   "dependencies": {
-    "nan": "^2.10.0",
-    "node-pre-gyp": "^0.10.3"
+    "nan": "^2.11.1",
+    "node-pre-gyp": "^0.11.0"
   },
   "devDependencies": {
     "bluebird": "^3.5.1",
     "commander": "^2.17.0",
     "murmurhash3js": "^3.0.1",
-    "node-pre-gyp-github": "1.3.1",
+    "node-pre-gyp-github": "^1.4.2",
```

Comment:
We have to stick to 1.3.x - https://github.com/bchr02/node-pre-gyp-github/issues/29

Reply:
OK, seems like my problem is not related to "node-pre-gyp-github" package, only to "node-pre-gyp", so I reverted "node-pre-gyp-github" to 1.3.1. Is it possible to accept PR in this case?

Reply:
Sure.